contextualized/regression/metamodels.py

```diff
@@ -23,11 +23,9 @@ def __init__(
         y_dim,
         univariate=False,
         encoder_type="mlp",
-        encoder_kwargs={
-            "width": 25,
-            "layers": 1,
-            "link_fn": LINK_FUNCTIONS["identity"],
-        },
+        width=25,
+        layers=1,
+        link_fn=LINK_FUNCTIONS["identity"],
```

Comment:
For function signatures you modify, can you also add type hints? This will make it easier to refactor in the future.